build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.hee.trainee.details"
-version = "0.24.5"
```

Comment:
I'd say this is a minor bump, rather than patch bump, since it's an enhancement and not a bug fix.